### DISCUSSION TITLE
Sidra token

### DISCUSSION THREAD

**Discussion by Fahadshah8284** [blockscout/blockscout#10617] [Q&A] [answered]
How someone stolen my sidra token please check it and get back my tokens 
![Screenshot_2024-08-19-13-41-37-22_40deb401b9ffe8e1df2f1cc5ba480b12](https://github.com/user-attachments/assets/bebea027-2785-454f-b4bc-31b94522681b)

**Comment by GimluCom** [comment, marked as answer]
This is the repo for the open-source Blockscout explorer. Please contact sidra directly about your issue.
